exporter/sapmexporter/README.md

```diff
@@ -10,6 +10,7 @@ another backend that supports the SAPM proto.
 format. It must be a full URL and include the scheme, port and path e.g,
 https://ingest.us0.signalfx.com/v2/trace. This can be pointed to the SignalFx backend or to
 another Otel collector that has the SAPM receiver enabled.
+- `api_endpoint` (no default): This is the endpoint where API calls will be made (such as for correlation). It must be a full URL (e.g. https://api.signalfx.com).
```

Comment:
This is slightly a nit, but in common usage an _endpoint_ usually includes the path and refers to the _http resource_ for an HTTP server.  I know other things in the collector are blurry on this distinction but it makes it easier for new users if the names of things follow standard terminology as much as possible.

Perhaps `api_base_url` or `api_url`.

Reply:
I researched this a bit more and it is pretty clear that when talking about APIs, esp. HTTP-based APIs, _endpoint_ is the full URL with path.  Endpoint can refer to a host in general in the context of lower-level protocols, but at the application layer, such as this, endpoint is more specific.

Reply:
Also I'm not sure I see a big distinction between a URL with and without a path. For the correlation stuff it can't be a path because the path is created dynamically based on the dimension being updated.

Though if for some reason somebody wanted to set api_endpoint to `http://host.name:1234/signalfx` that proxied to `api.signalfx.com` I don't think the fact that one has a path and the other doesn't makes one an endpoint and one not.

Reply:
Yeah it's not about having a path or not, it's about calling an API base URL an endpoint.  A base URL can include a path if it is needed for a proxy.  It has to do with the fact that the correlation stuff is constructing paths and appending them to this URL provided.